contracts/kip13/InterfaceIdentifier.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.4.24;
```

Comment:
Please refer `klaytn/klaytn/contracts/generate.go` and add a script to generate `contracts/kip13/InterfaceIdentifier.go`.

Reply:
Modified version and added the script to generate. 77ac2e1252b8a32516a449af795935c1e164bb6a